### DISCUSSION TITLE
Venusian Marshal (BAUHAUS) [U]

### DISCUSSION THREAD

**Discussion by Michalx11** [secretcowlevel/DoomtrooperPublic#39] [Card Balance]
Marshal's ability is boring plus stats are too high.
Here is a suggestion:

Venusian Marshal
6/5/6/6
Whenever you play NON-UNIQUE Baushas warrior you may pay additional 2DP. If you do that warrior gains SLAYER until the beginning of your next turn. If you pay 4DP that warrior gains SLAYER+FIRST STRIKE until beginning of gi our next turn

**Comment by TomekPL-1** [comment]
Etoiles Mortant (BAUHAUS) [C]
ETOILESMORTANT
Enter your SQUAD with one counter. Once during your turn you can remove one counter to gain until end of turn one option:
-POWERIMMUNITY
-BLESSED
-Gains +3 to [F], [S] and [A] when combating DARKLEGION Warriors and HERETICs.
F: 4 S: 2 A: 4 V: 4

Venusian Marshal (BAUHAUS) [U]
When he enter your Squad you can add one counter on every warrior. Once per turn as one ACTION he can perform one from below action:
-Your other non-UNIQUE Warriors gain +2 to [F] , [S] and [A] until beginning of your nex turn.
-Put one counter on target Warrior
F: 6 S: 5 A: 6 V: 7

VENUSIAN RANGER Bauhaus [U]
VENUSIANRANGER 
Enter your SQUAD with one counter.
Once during your turn you can remove one counter to chose one option:
-During next combat gain +2 to their [F] , [S] , and [A] (this is cumulative)
-During next combat gain FIRST STRIKE
 F: 4 S: 3 A: 4 V: 4

Max Steiner Bauhaus [UR]
UNIQUE 
When your Venusian Rangers become WOUNDED put one counter on him and he have +5 to [F] and [S] until healed
 F: 4 S: 7 A: 9 V: 8

**Reply by Sol-au-fein** [reply]
This is exceptionally bad design. 
It all depends on Venusian Marshall in play and allows only one possible game style.
You need to spend actions to distribute tokens, while other factions dont have to. 
Less actions to play supports and equipment.
If you dont have Marshall in play or dont spend actions to distribute tokens, your warriors are just empty cards with statline and utterly useless. 
I really hope i am not only one who see how mind-numbingly flawed this suggestion is. All you will do is counting tokens and activate your warriors endlessly. 
If this gets actually ingame, im leaving that same day. I dont mind game breaking bugs as much as i mind this. The fact you dont see consequences of your idea, however experienced player you might be, is shocking.

**Reply by TomekPL-1** [reply]
You can add card similar to Cybertronic 'Scrap Heap' adding counters. Redesigning a little Aide Camp to have similar  possibility.

Venusian Rangers also gain counters from Max Steiner

**Reply by TomekPL-1** [reply]
And add Bauhaus COP I mentioned before with counters ability, which make Bauhaus Faction game style Unique like Cybertronic Augments

**Reply by Michalx11** [reply]
@Sol-au-fein Counters are used in most digital CCG games. They can be very effective strategy.
It is ok to comment and give feedbacks on other players suggestions but insulting them is NOT! 
Please refrain from toxic language in the future

**Reply by Sol-au-fein** [reply]
I thought as much you will side against me. Have you even read what i said? There is no toxic language. 
Your comment about tokens is irrelevant. 
I have made my point.

**Reply by framerate** [reply]
@Sol-au-fein Maybe its a language barrier, but

> This is exceptionally bad design.

and

> I really hope i am not only one who see how mind-numbingly flawed this suggestion is.

comes across as insulting and toxic to many reading it. Just try and understand that our goal is not to fight or insult each other here. Our goal is to make Doomtrooper the best game we can! And you contribute a lot and it is appreciated!

So try and use words like "I'm not sure I think this is good design" or "This design may be flawed" followed by "here's an example" and I think it will help you get your point across and continue the discussion 👍

**Comment by TomekPL-1** [comment]
Based on the feedback I propose slighty improved Bauhaus couners

Etoiles Mortant (BAUHAUS) [C]
ETOILESMORTANT
Enter your SQUAD with one counter. During your turn you can remove counters to gain until beginning of your next turn below option, each option can be selected only once per turn:
-POWERIMMUNITY
-BLESSED
-Gains +3 to [F], [S] and [A] when combating DARKLEGION Warriors and HERETICs.
F: 4 S: 2 A: 4 V: 4

Venusian Marshal (BAUHAUS) [U]
When he enter your Squad you can add one counter on every warrior. Once per turn he can perform one from below action:
-Your other non-UNIQUE Warriors gain +2 to [F] , [S] and [A] until beginning of your nex turn.
-Put one counter on target Warrior
F: 6 S: 5 A: 6 V: 7

VENUSIAN RANGER Bauhaus [U]
VENUSIANRANGER
Enter your SQUAD with one counter.
You can remove one counter to chose:
-During next combat gain +2 to their [F] , [S] , and [A] (this is cumulative)
-During next combat gain FIRST STRIKE
F: 4 S: 3 A: 4 V: 4

Max Steiner Bauhaus [UR]
UNIQUE
When your Venusian Rangers become WOUNDED put one counter on him and he have +5 to [F] and [S] until healed
F: 4 S: 7 A: 9 V: 8

Aide-De-Camp Bauhaus [C]
COMMANDER 
Assign another non UNIQUE Warrior to the Aide-de-Camp. That card and copies of that card in your squad gain +3 to [F] , [S] , [A] and [V] . At the beginning of your turn he put 1 counter on assigned warrior.
 F: 3 S: 4 A: 7 V: 7

**Reply by Sol-au-fein** [reply]
Hey @TomekPL-1  i think your design has serious flaws. 
This is not better. You completely missed my point. You only creating huge disadvantage.
You need to invent new cards in order to be even base warriors functional? That is wrong approach.
If you want to add new cards, make them to enhance existing strategies without redesigning whole faction.

Here is what i suggest:
Leave out counters idea.
Leave Etoiles Mortants and Venusian Rangers as good as they are. There is no need for change.
There is good functional strategy with Etoiles Mortants and Duval.
There is no good strategy for Rangers. If something can be changed for better, it would be Max Steiner to create some viable strategy for them.
Leave Aide-de-Camp as commander as other factions have. it will work with both strategies.

Im fine with adding Bauhaus Cop to pose as misscom for Bauhaus.

**Reply by TomekPL-1** [reply]
I agree to disagree with you about counters idea. Would be also good to hear what other peoples think about this.

**Reply by 21-ric** [reply]
I'll try be be diplomatic with the language barrier.
I agree with @Sol-au-fein .
> Here is what i suggest:
> Leave out counters idea.
> Leave Etoiles Mortants and Venusian Rangers as good as they are. There is no need for change.
> There is good functional strategy with Etoiles Mortants and Duval.
> There is no good strategy for Rangers. If something can be changed for better, it would be Max Steiner to create some viable strategy for them.
> Leave Aide-de-Camp as commander as other factions have. it will work with both strategies.

Evryone has the right to suggest an idea. 

I prefer the original card :
8/7/8/8 For each Venusian Marshal in your Squad, all your non-personality Bauhaus Doomtroopers (but not Venusian Marshals) gain +2 to F, S and A.

**Reply by Michalx11** [reply]
"The future belongs to brave and bold ones".
Counters are really cool. I love playing with them in MTG. They are much easier to implement in digital CCG than paper. 
Counters in Doomtrooper arent widely used so i undertand some players are reluctant to use them but I dont rhink we should be afraid of them. Tomek's idea is bold and fresh and it could work nicely as Bauhaus warriors could easily adapt to multiple strategies and thats super powerful.
The only thmg I worry about is time. This kind of overhaul will consume considerable amount of programming which we are short of. If time allows i would definitely love to see counter idea like this in DT as it could spruce up the game and add a variety in strategies.
If there is no time to implement counters then at the very least I suggest to change Max and Venusian Marshal. Their abilities are too boring with stats bonuses... I gave suggestion for both... plus Venusian Marshal's stats are waaay too high as non UNIQUE warriors shouldnt be stronger than Max and Valerie. Bauhaus Cop or Eva Valmonte could be added (if time allows if course) as counter spells

**Reply by TomekPL-1** [reply]
Yeah. I hope that Justin will find time to implement counters or something similar. Bauhaus faction deserve to have cool mechanics not just stat boost (which is really boring).

**Comment by Sol-au-fein** [comment]
`Venusian Marshalls are sent out on missions too harsch or too difficult to take on by foot or vehicle. They ride sturdiest of mounts - bred to endure those harsch conditions. If it is your dream to wear legendary white skull facemask of the rangers - service as Marshall is sure to wake you up.
The Venusian Marshall rides a horse. There is one statline for both horse and the rider to show how hard is to discern the two on the battlefield.
Special rule: Terrain Jumping.
Equipment: Kampfkanone and Violator sword.`

What i did notice from his warzone stats, he is significantly better at shooting than close combat fighting.
There is nothing about inspiring or buffing other troops.
